README.md

```diff
@@ -0,0 +1,17 @@
+# Flight
+
+WIP
+
+To generate the sprite file, [sprite.svg](ember-flight-icons/public/icons/sprite.svg), follow the instructions in the README of [edenspiekermann/sprite.sh](https://github.com/edenspiekermann/sprite.sh) e.g.
+
+- `npm install spritesh -g`
+- `cd ember-flight-icons/public/icons` so that the `sprite.svg` file will be created in the right directory
+- `spritesh`
```

Comment:
Take it or leave it comment: could recommend `npx spritesh` here and avoid needing the step in Line 7

Reply:
👍 cool TIL, thanks!